app/controllers/events_controller.rb

```diff
@@ -27,6 +27,18 @@ def index
     end
   end
 
+  # GET /events/calendar
+  # GET /events/calendar.js
+  def calendar
+    @bioschemas = @events.flat_map(&:to_bioschemas)
+    # now customize the list by moving all events longer than 3 days into a separate array
+    @long_events, @events = @events.partition { |e| (e.end - e.start) > 3.days }
```

Comment:
I think 5 days might be more reasonable. Should be configurable also.

Reply:
done